packages/utilities/fast-react-wrapper/src/index.ts

```diff
@@ -310,6 +311,8 @@ export function reactWrapper(
             }
         }
 
+        ReactComponent.displayName = getTagName(type, config);
```

Comment:
Were we going to remove this instance in favor of adding it only for the forwardRef instance?